### ISSUE TITLE
Missing data file: weather/train_data_x.npy

### ISSUE THREAD

**Issue by guangyuyang001** [SaberaTalukder/TOTEM#5]
I tried to run the forecasting script(forecasting/scripts/all.sh), but got this error: No such file or directory: 'forecasting/data/weather/train_data_x.npy'
Could you provide this file or instructions to generate it?

**Comment by SaberaTalukder** [comment]
All the data is located [here](https://drive.google.com/drive/u/0/folders/1gI36rS8irRZ32ibzKBPGncDmMXQtEf1C), step 2 of the README. Was there a problem in downloading it?

**Comment by guangyuyang001** [comment]
Thank you for your helpful explanation! I really appreciate it.
I'm very interested in the ablation from Figure 8: Discrete Token vs. Patches with MLP. Could you kindly share the code for that part?

**Comment by guangyuyang001** [comment]
When I try to run the script using the provided instructions, I encounter the following error:
     No such file or directory: 'add/path/to/trained/vqvae'
I attempted to replace it with what I believe to be the correct path:
    /forecasting/saved_models/electricity/CD64_CW256_CF4_BS4096_ITR15000
However, this also leads to the same error:
    No such file or directory: '/forecasting/saved_models/electricity/CD64_CW256_CF4_BS4096_ITR15000'
Could you kindly clarify what the correct path should be, or whether the pre-trained VQVAE model needs to be downloaded or trained separately before running the code?
Thank you for your time and help.

**Comment by SaberaTalukder** [comment]
The code for the MLP should be simple to reproduce see page 11 of the [paper](https://arxiv.org/pdf/2402.16412) "The MLP has 3-layers ReLU activations, uses dropout with p = 0.1 after the second layer, and concludes with a layernorm; this architecture is modeled after similar architectures in the literature like Das et al. (2023a). The patch-based MLP takes in an uncompressed time series."

My understanding is that you're trying to run this [electricity forecasting script](https://github.com/SaberaTalukder/TOTEM/blob/master/forecasting/scripts/electricity.sh); for this, you will need to train your own VQVAE or download the pretrained one (only the [generalist](https://drive.google.com/drive/u/0/folders/1TSwPHDMAhcpe2AKl4xsVbUUmAvd_Tp-Z) VQVAEs are released, not the specialists which is what the electricity forecasting script is looking for). They are pretty fast to train though.
